core/tellie_server.py

```diff
@@ -516,8 +516,24 @@ def read_pin_sequence(self):
         if len(numbers) == 0:
             self.log_phrase("Sequence doesn't appear to have finished..", 0, _snotDaqLog)
             return None
-        elif len(numbers) == 2:
+        #If we only get one number it is likely the PIC hasnt finished writing to the buffer
+        if len(numbers) == 1:
+            #Wait a bit longer and then read out the buffer again
```

Comment:
I think this wants to be in a try catch incase there is a case where the PIC outputs a single number, but isn't in the middle of a sequence.